R4

arscan opened the pull request:
We should wait until we are stable on Crucible with this version of plan_executor and have had a chance to do some testing through that across a number of servers.  Progress here: https://github.com/fhir-crucible/crucible/pull/304

arscan committed:
Move all STU3 references to FHIR::STU3

arscan committed:
Make resource generator tests pass.

arscan committed:
More r4 updates.

jawalonoski committed:
Fix broken resource generator unit tests.

jawalonoski committed:
Fix resource generator for STU3 DataElement.

arscan committed:
Fix FHIR structure in R4.

arscan committed:
Various r4 fixes.

arscan committed:
Update README.

arscan commented:
Test Suites on R4 appear to be stable, and will be deployed to Crucible shortly.  Holding off on merging to master because R4 TestScript implementation hasn't been checked yet -- it is likely that something will need to be updated.  FYI @jawalonoski 

@tstrass @mgramigna -- do you have a need for R4 TestScript testing?

mgramigna commented:
Currently our TestScript testing is focused on stu3, but we will eventually need support for R4 with the TestScripts.

So yes we will eventually have that need, though it is not time-sensitive.

EDIT: We are waiting on the QICore IG for R4 to be released before we move on to R4 for TestScript work.

arscan committed:
R4 ImplementationGuide generator incorrect fhirVersion cardinality

arscan committed:
RelatedPerson generates invalid R4 resource.

Bownairo committed:
Fix a typo in Sprinkler Search Test

The UCUM url is unitsofmeasure.org not unitofmeasure.org.

jawalonoski committed:
Merge pull request #180 from Bownairo/ucum-typo

Fix a typo in Sprinkler Search Test

arscan committed:
Merge branch 'master' into r4

arscan committed:
Merge pull request #176 from fhir-crucible/r4-relatedperson-gen

RelatedPerson generates invalid R4 resource.

arscan committed:
Merge pull request #174 from fhir-crucible/r4-ig-gen-fix

R4 ImplementationGuide generator incorrect fhirVersion cardinality

arscan committed:
Fix some language and narrative generation.

arscan committed:
Update gemfile.lock

pedro-w committed:
Create a patient on the client, if needed
Also use the standard 'minimal_patient' resource instead
of an ad-hoc definition

pedro-w committed:
Use a patient resource fixture with text in it.

arscan committed:
Merge pull request #190 from pedro-w/read-test-fix

Read test fix
